core/src/test/java/org/elasticsearch/ExceptionSerializationTests.java

```diff
@@ -758,7 +757,7 @@ public void testIds() {
         ids.put(92, org.elasticsearch.indices.recovery.DelayRecoveryException.class);
         ids.put(93, org.elasticsearch.search.warmer.IndexWarmerMissingException.class);
         ids.put(94, org.elasticsearch.client.transport.NoNodeAvailableException.class);
-        ids.put(95, org.elasticsearch.script.groovy.GroovyScriptCompilationException.class);
+        ids.put(95, null);
```

Comment:
Yes, see the note in ElasticsearchException. numbers should never be removed, only assigned to null if the exception is removed.